### ISSUE TITLE
New block (Essential)

### ISSUE THREAD

**Issue by Zeuscp1** [tsunamayo/Starship-EVO#1990]
Hello, we are missing a glass piece, that would fit here 

https://ibb.co/g9RschF

Basically an inverted triangle piece.

**Comment by tsunamayo** [comment]
Yes the hepta windows, it was already suggested. You can use a tetra put on the other side for now, it will stick out a bit but you will be able to continue to build. Thanks
